test/test.js

```diff
@@ -180,19 +175,65 @@ describe('YeldaChat', () => {
     })
   })
 
+  describe('yeldaChat.voiceFirstUI', () => {
+    it('should contain voiceFirstUI class', () => {
+      // Reset the DOM
+      yeldaChat.unLoadChat()
+
+      // resets previous response
+      mock.reset()
+      // Mock JSON response
+      mock.get('https://app.yelda.ai//assistants/12345678/chatBubble/fr_FR', {
+        status: 201,
+        body: JSON.stringify({ data: {isVoiceFirstUI: true} })
+      })
+
+      const mockData = {
+        'assistantSlug': 'testClient',
+        'assistantId': '12345678',
+        'chatPath': 'chat'
+      }
+
+      yeldaChat.setupChat(mockData)
+      // setTimeout used to wait the for the mock response
+      setTimeout(() => {
+        expect(yeldaChat.iframeContainer).to.have.attribute('class', 'yelda_iframe_container voiceFirstUI y_active')
+      })
+    })
+
+    it('expect assistantImage not exists in document', () => {
+      expect(yeldaChat.assistantImage).to.be.null
+    })
+
+    it('should not contain assistant image', () => {
+      expect(document.querySelector('#yelda_assistant_img')).to.be.null
+    })
```

Comment:
Same here : whatever i do, the test pass whereas it should not
```
// test passed
 if (isVoiceFirstUI) {
        this.openChat()
        this.webChatContainer.appendChild(this.assistantImage)
        return
      }
// test passed
 if (isVoiceFirstUI) {
        this.iframeContainer.classList.add('voiceFirstUI')
        this.openChat()
        this.assistantImage = 'ntull'
        return
      }
```

Can you recheck ?